packages/core/http/core-http-server-internal/src/csp/config.ts

```diff
@@ -83,6 +83,13 @@ const configSchema = schema.object(
     report_to: schema.arrayOf(schema.string(), {
       defaultValue: [],
     }),
+    report_only: schema.maybe(
+      schema.object({
+        form_action: schema.arrayOf(schema.string(), {
+          defaultValue: [],
+        }),
+      })
+    ),
```

Comment:
NIT: you don't need to add the `maybe` modifier to `report_only`, given its only property has a default value

```ts
report_only: schema.object({
    form_action: schema.arrayOf(schema.string(), {
        defaultValue: [],
     }),
}),
```

Reply:
@pgayvallet By the way, do we want `report_only` or `reportOnly`? I see that for the part of CSP configuration that is directly serialized to a policy, we use `snake_case` since it's closer to the `kebab-case` that the "raw" CSP syntax requires. However, for anything else that is not directly serialized to a policy, we use `camelCase`, as for any other Kibana configuration entries that aren't bound to any external naming convention.

Reply:
Discussed with @pgayvallet over Slack - Pierre is leaning towards keeping `snake_case` as it's still close to real CSP directive even though not 100%. Sounds good to me, let's keep it as is.